expat/doc/reference.html

```diff
@@ -1808,10 +1808,11 @@ <h4 id="XML_SetElementDeclHandler">XML_SetElementDeclHandler</h4>
 </pre>
 <p>Sets a handler for element declarations in a DTD. The handler gets
 called with the name of the element in the declaration and a pointer
-to a structure that contains the element model. It is the
-application's responsibility to free this data structure using
-<code><a href="#XML_FreeContentModel"
->XML_FreeContentModel</a></code>.</p>
+to a structure that contains the element model. It's the user code's 
+responsibility to free model when finished with it. See <code>
+<a href="#XML_FreeContentModel">XML_FreeContentModel</a></code>.
+There is no need to free the model from the handler, it can be kept
+around and freed at a later stage.
```

Comment:
There is a trailing `</p>` here, as confirmed by the CI at https://github.com/libexpat/libexpat/runs/5393482973?check_suite_focus=true .